dev/src/dev/migrate.clj

```diff
@@ -101,3 +106,35 @@
 
       (throw (ex-info "Invalid command" {:command cmd
                                          :args    args})))))
+
+(defn- change->sql
+  [change sql-generator-factory database]
+  {:forward (str/join "\n" (for [stmt (.generateStatements change database)
+                                 sql (.generateSql ^SqlGeneratorFactory sql-generator-factory stmt database)]
+                             (.toString sql)))
+   :rollback (str/join "\n" (for [stmt (.generateRollbackStatements change database)
+                                  sql (.generateSql ^SqlGeneratorFactory sql-generator-factory stmt database)]
+                              (.toString sql)))})
```

Comment:
Not a train smash for just doing it twice, but an inner helper function could really help bring down the density and noise.

```clojure
{:forward  (statement->sql (.generateStatements change db))
 :rollback (statement->sql (.generateRollbackStatements change db))}
```